baselines/acer/acer_simple.py

```diff
@@ -13,6 +14,8 @@
 from baselines.a2c.utils import get_by_index, check_shape, avg_norm, gradient_add, q_explained_variance
 from baselines.acer.buffer import Buffer
 
+import os.path as osp
```

Comment:
that's funny, looks like this fix https://github.com/openai/baselines/commit/569bd426297859c57c0c2ba6482f1349ac12fc01 missed the import?

Reply:
I think so... did not seem to work without it

Reply:
Closes https://github.com/openai/baselines/pull/312

Reply:
sounds good, but we'll have to export this to baselines.